.gitignore

```diff
@@ -17,3 +17,5 @@ docs/_static/Tutorial_*.png
 docs/generated
 poetry.lock
 references
+
+.vscode/
```

Comment:
This one should be after `.mypy_cache`, we probably don't need the last slash too.

Reply:
Fine by me, bonus points for the `.` to hide it! :)